app/grandchallenge/reader_studies/templates/reader_studies/display_set_create.html

```diff
@@ -39,11 +39,9 @@ <h2>Add {{ type_to_add }} to {{ object }}</h2>
 {% block script %}
     {{ block.super }}
     {{ form_media }}
-    <script src="{% static 'reader_studies/js/display_set_create.js' %}" type="text/javascript"></script>
-    <script src="{% static 'reader_studies/js/autocomplete_htmx.js' %}" type="text/javascript"></script>
+    <script type="module" src="{% static 'components/js/create_extra_interfaces.mjs' %}" type="text/javascript"></script>
+    <script type="module" src="{% static 'components/js/autocomplete_htmx.mjs' %}" type="text/javascript"></script>
     <script type="text/javascript" src="{% static "cases/js/ignore_enter_button.js" %}"></script>
-
-    {# TODO: This might not be used #}
     {% include 'grandchallenge/partials/drf_csrf.html' %}
```

Comment:
This partial is definitely necessary?

Reply:
Actually not, also not in the update html. In the list view template it is necessary for removing display sets.